temporian/implementation/numpy/data/dtype_normalization.py

```diff
@@ -141,13 +145,16 @@ def _str_to_bytes(feat_array: np.ndarray) -> np.ndarray:
 
     # Convert np.datetime -> np.float64
     if array_dtype == np.datetime64:
+        logging.debug("From np datetime64")
         # nanosecond resolution as in timestamps
         feature_values = datetime64_array_to_float64(feature_values)
 
     # Convert np.object_, np.str_ -> np.bytes_
     elif array_dtype == np.str_:
+        logging.debug("From np str_")
```

Comment:
super nit, saying np.datetime64 and np.str_ would make it clearer IMO (with the dots)

Reply:
Good point.
*Done (here and in other places)